public/app/features/transformers/editors/HistogramTransformerEditor.tsx

```diff
@@ -15,22 +14,20 @@ export const HistogramTransformerEditor: React.FC<TransformerUIProps<HistogramTr
   const labelWidth = 18;
 
   const onBucketSizeChanged = useCallback(
-    (evt: FormEvent<HTMLInputElement>) => {
-      const val = evt.currentTarget.valueAsNumber;
+    (val?: number) => {
       onChange({
         ...options,
-        bucketSize: isNaN(val) ? undefined : val,
+        bucketSize: Number.isNaN(val) ? undefined : val,
```

Comment:
```suggestion
        bucketSize: val,
```
not needed because the input is already checked